packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelect.tsx

```diff
@@ -15,6 +15,7 @@ export type SingleEntitySelectProps = {
 
 export const SingleEntitySelect = ({
   disableBackgroundBlur = false,
+  dropdownPlacement,
```

Comment:
I took the liberty of adding this prop and passing it down to `SingleEntitySelectMenuItemsWithSearch`. I don't know if it's a good idea:
- it does expand `SingleEntitySelect` contract... which I recon can be less maintainable.
- however the `dropdownPlacement` is already a prop available in `SingleEntitySelect` type interface; it just wasn't passed down to the underlying component, which I found was weird
- it allows testing and using this prop in _Storybook_


Let me know what you think.

Reply:
Yes makes sense to pass it if the below component has it I agree!